docker/web/build.gradle

```diff
@@ -12,7 +12,7 @@ dependencies {
     js project(path: ':web', targetConfiguration: 'js')
 }
 
-evaluationDependsOn Docker.registryProject('nginx-base')
+evaluationDependsOn Docker.registryProject('nginx-noroot-base')
```

Comment:
do we have any references left to nginx-base, or is it safe to either remove nginx-base from the registry, or reuse nginx-base (and change its imageName etc) instead of adding a new proj to the registry?

Reply:
If there are references left, I can remove them, however I thought "we probably have a bunch of documentation which includes the old version, we should leave removing it until after users have had a migration period of at least one release".

I can open a ticket for that?

Reply:
I don't _think_ we consider those internal images as being part of our public api/artifacts, so removing all traces of them should be safe.

@devinrsmith can you confirm?

Reply:
I'll let @devinrsmith but I'm in agreement, should trash nginx-base

Reply:
The port number we use needs to be changed everywhere.  Code, documentation, any published works, etc. It's a lot of extra work :-/

Reply:
I looked in deephaven.io and can't see anywhere obvious that has docker-compose yaml to edit for the updated port...

@margaretkennedy any ideas?

Reply:
I looked for `docker-compose` and `port` and found nothing to update

Reply:
> I looked in deephaven.io and can't see anywhere obvious that has docker-compose yaml to edit for the updated port...
> 
> @margaretkennedy any ideas?

We don't include the actual yaml anywhere in the user guide. Sounds like any documentation updates would be in the file itself or in deephaven-core's README, if even necessary?

Reply:
Nothing references that port, it isn't exposed to users at all. Instead, only the envoy proxy talks to it, the only place to update is the envoy yaml (which I think you are doing anyway). Envoy acts as a reverse proxy in front of nginx, and there is no other way to connect to it.